src/main/java/ev3dev/actuators/ev3/EV3Led.java

```diff
@@ -109,19 +108,22 @@ private void checkPlatform() {
      */
     @Override
     public void setPattern(final int pattern) {
-        //Off
+
+        final String off = Integer.toString(0);
+        final String on = Integer.toString(255);
```

Comment:
If we want to change this, it might be worth it to also have only a single `writeString` call below for each LED. Each if branch would only set the value to pass to `writeString()`.

Reply:
I kind of want to let the kids play with the integer settings. But I think the best they can do is red + green = some kind of glowing brown. An ambitious kid could have the colors fade instead of change suddenly. Not this fix.

Reply:
I meant this only about the code style - there should be no problem with adjusting the integer value later. It's just that the code below this line that calls `writeString()` looks repetitive and it might be better to join these calls into just two calls at the end of the if-ladder.

Reply:
I'll give that a try tonight and see what it looks like.